src/systems/camera-system.js

```diff
@@ -410,9 +401,9 @@ export class CameraSystem {
     const translation = new THREE.Matrix4();
     let uiRoot;
     return function tick(scene, dt) {
-      this.viewingCamera.object3DMap.camera.matrixNeedsUpdate = true;
-      this.viewingCamera.object3DMap.camera.updateMatrix();
-      this.viewingCamera.object3DMap.camera.updateMatrixWorld();
+      this.viewingCamera.matrixNeedsUpdate = true;
+      this.viewingCamera.updateMatrix();
+      this.viewingCamera.updateMatrixWorld();
```

Comment:
Interesting code here... I'm guessing I wrote this out of pure frustration

Reply:
Yeah I didn't want to change that in this PR. This camera system code is definitely worth a revisit in general. I think we should probably have multiple THREE.PerspectiveCamera instances rather than trying to mutate a single one and not clobber previous state.